academy/3-ibc/2-connections.md

```diff
@@ -97,6 +97,8 @@ Establishing an IBC connection (for example, between chain A and chain B) requir
 3. OpenAck
 4. OpenConfirm
 
+<YoutubePlayer videoId="E3ZvqdY2tL8" />
+
 A high level overview of a successful four-way handshake is as follows:
 
 ### Handshake 1 - OpenInit
```

Comment:
I would also maybe call it `Handshake step 1`...

Reply:
Good catch, will change